src/php/ext/grpc/channel.c

```diff
@@ -509,27 +533,72 @@ void php_grpc_delete_persistent_list_entry(char *key, php_grpc_int key_len
 static void php_grpc_channel_plink_dtor(php_grpc_zend_resource *rsrc
                                         TSRMLS_DC) {
   channel_persistent_le_t *le = (channel_persistent_le_t *)rsrc->ptr;
+  if(le == NULL) {
```

Comment:
if [space]

Reply:
Done.
Also update other places with "if [space]" and "} [space] ("